lib/testing/src/lib/content-services/actions/upload.actions.ts

```diff
@@ -0,0 +1,84 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { browser } from 'protractor';
+import * as path from 'path';
+import * as fs from 'fs';
+
+export class UploadActions {
+
+    async uploadFile(alfrescoJsApi, fileLocation, fileName, parentFolderId) {
```

Comment:
Maybe we pass the "alfrescoJsApi" in the constructor? So that we don't need to pass it for every method? @eromano what do you think?

Reply:
yep @DenysVuika you are right will better change it. We did it for all the other service for example @alexa-fabian  you can give a look to the query.service.ts

Reply:
suggested changes are now completed